document_page/wizard/document_page_create_menu.py

```diff
@@ -52,9 +50,9 @@ def document_page_menu_create(self):
         # only the super user is allowed to create menu due to security rules
         # on ir.values
         # see.: http://goo.gl/Y99S7V
-        action_id = obj_action.sudo().create(value)
+        action_id = obj_action.with_user(SUPERUSER_ID).create(value)
 
-        menu_id = obj_menu.sudo().create(
+        menu_id = obj_menu.with_user(SUPERUSER_ID).create(
```

Comment:
Why are you replacing sudo()?

Reply:
It was not necessary to do that so I left it as it was before